tools/diar_eval.py

```
@@ -0,0 +1,477 @@
+    ein `open(..., "w")` und ueberschriebe echtes Material. Der wichtigste Constraint dieses
+    Plans („nie nach projekte\\ schreiben") war damit allein durch Disziplin gesichert, und ein
+    Vertipper reicht. `os.path.realpath` auf BEIDEN Seiten, sonst kaeme ein Symlink daran vorbei.
+    """
+    ziel = os.path.realpath(pfad)
+    wurzel = os.path.realpath(EVAL)
+    if ziel != wurzel and not ziel.startswith(wurzel + os.sep):
+        raise SystemExit(f"Ziel liegt ausserhalb von eval/: {ziel}")
+    return ziel
+
+
+def _projekte_standard():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_projekte_standard`

(ANN202)